src/Domain/Structure/DirectionalId.hpp

```diff
@@ -16,7 +16,7 @@ class er;
 /// \endcond
 
 template <size_t VolumeDim>
-struct DirectionId {
+struct DirectionalId {
```

Comment:
sorry I missed this on my last review. Could you document this a bit so it shows up in doxygen?

Reply:
done